src/main/java/WayofTime/alchemicalWizardry/client/nei/NEIMeteorRecipeHandler.java

```diff
@@ -36,14 +36,24 @@ public CachedMeteorRecipe(MeteorParadigm meteor) {
             this.input.add(new PositionedStack(meteor.focusStack, 74, 4));
             int row = 0;
             int col = 0;
+            
+            float totalMeteorWeight = 0.0f;
+            for (MeteorParadigmComponent component : meteor.componentList) {
```

Comment:
unify the loop below

Reply:
Can't change it to one loop because the first iteration already needs the total weight, changed it to a function call